cvat/requirements/base.txt

```diff
@@ -43,6 +43,7 @@ scikit-image==0.15.0
 tensorflow==1.15.0
 keras==2.2.5
 opencv-python==4.1.0.25
+opencv-contrib-python==4.1.2.30
```

Comment:
Only one package should be installed.
Installing both packages is leading to conflicts.